### DISCUSSION TITLE
Server browser on Epic version does not display ping

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13973] [Bug reports]
### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/37850646/3818f0d6-3542-49e7-80ad-c8ffeaaacb2a)

### Reproduction steps

1. Launch Barotrauma™ on Epic™

2. Observe no ping in server browser
### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Why are there only numbers in the name column?

**Reply by Vykn** [reply]
it's sorted by alphabetical.

**Comment by Vykn** [comment]
I have become aware that Epic has no ability to measure ping. I still think it's a bug that the column shows up at all (it looks really silly that the fields are all missing) but considering that nobody plays Barotrauma™ on Epic™, I would understand if this gets wontfix'd.
